src/server/handlers/participants/{Type}/{ID}.ts

```diff
@@ -0,0 +1,78 @@
+import { Request, ResponseToolkit, ResponseObject } from '@hapi/hapi';
+import { Context } from '~/server/plugins';
+import { retrieveConsent, createConsent, updateConsent, deleteConsent } from '~/domain/participants';
+import { Schemas } from '@mojaloop/api-snippets/lib/v1_1';
+import { Consent } from '~/model/consent';
+import * as Types from '~/interface/types'
+import { IDTypeNotSupported } from '../../../../model/errors';
+import Boom from '@hapi/boom';
+
+
+export async function get (
+  _context: Context,
+  request: Request,
+  h: ResponseToolkit): Promise<ResponseObject> {
+  if (request.params.Type !== 'CONSENT') {
+    return Boom.boomify(new IDTypeNotSupported())
+  }
+
+  const consentId = request.params.ID
+  const consent = await retrieveConsent(consentId)
```

Comment:
do we really want the synchronous interface here? should we really want to wait until the promise resolve and not deliver the result via async PUT call?